packages/mui-base/src/SelectUnstyled/SelectUnstyled.types.ts

```diff
@@ -1,31 +1,48 @@
 import * as React from 'react';
 import { DefaultComponentProps, OverrideProps, Simplify } from '@mui/types';
-import { SelectOption, UseSelectButtonSlotProps, UseSelectListboxSlotProps } from '../useSelect';
+import {
+  SelectOption,
+  SelectValue,
+  UseSelectButtonSlotProps,
+  UseSelectListboxSlotProps,
+} from '../useSelect';
 import PopperUnstyled, { PopperUnstyledProps } from '../PopperUnstyled';
 import { SlotComponentProps } from '../utils';
 
 export interface SelectUnstyledRootSlotPropsOverrides {}
 export interface SelectUnstyledListboxSlotPropsOverrides {}
 export interface SelectUnstyledPopperSlotPropsOverrides {}
 
-export interface SelectUnstyledCommonProps {
+export interface SelectUnstyledOwnProps<TValue extends {}, Multiple extends boolean> {
```

Comment:
```suggestion
export interface SelectUnstyledOwnProps<TValue extends {}, Multiple extends boolean = false> {
```
I see that some places have a default type but some are not. Should they all have the default type `= false`?

I tested and I think having a default type gives a better DX https://www.typescriptlang.org/play?ts=4.0.5#code/JYWwDg9gTgLgBAJQKYEMDG8BmUIjgcilQ3wG4BYAKCqrQgDsBneAWQFcAbGYAZSQ6QY4AXjgAeACoAaOCzhIAHjCT0AJozgAjCBAEp6IuJhQdGSAHwAKMDjCMAXHADecEJ25gBAfkctScADcTNiRfeSUVdTgYKBC4LzgJAG0AXThHCX8GAGEAC30AcyQfOEt6JAB3ADVg0Nlw5TUNGLiE5LSMgEoRc0CIYFU4AF9u4V6nKjg4Ihg2KAN6Tg4qIZpKMXYuXn5BeCCOEOEnJJcBxwBGYZShuBz8+iKjssqag6RR8aGbgHpzNbomKx3NsBBgAEyGSQyOSKRpRbS6VD0Kw2CB2RwuNxbTzFXz+fYhMKwyLNWJIeKJVLpRJZeh5Qq40rlaq1IkRJrRMkU9rUiQfPoDYb8iaUKYzOYLJYrNYbYF8UEwCEEpBHE5wM5wS5Da63On3R5OZ4st7Cr5wX5AA.

Reply:
@siriwatknp In the Autocomplete (from which this solution was inspired), only the component has default values for generic parameters. It makes sense, as the component actually defines the default value for the `multiple` prop if it's not defined.
In the example you posted, it's also the component that has the default generic value (similarly to `SelectUnstyledType` from my PR)